pyvortex/src/array.rs

```diff
@@ -477,7 +477,7 @@ impl PyArray {
     ///
     ///     >>> a = vortex.array(['a', 'b', 'c', 'd'])
     ///     >>> a.slice(3, 3).to_arrow_array()
-    ///     <pyarrow.lib.StringArray object at ...>
+    ///     <pyarrow.lib.StringViewArray object at ...>
```

Comment:
I think we have to revert this, it's too weird for now

Reply:
I think we should keep it, otherwise it strange only some compute functions can change encoding